src/rules/selector-pseudo-element-no-unknown/README.md

```diff
@@ -0,0 +1,93 @@
+# selector-pseudo-element-no-unknown
+
+Disallow unknown pseudo-element selectors.
+
+```css
+    a::before {}
+/**    ↑
+ * This pseudo-element selector */
+```
+
+All vendor-prefixes pseudo-element selectors are ignored.
+List of valid pseudo-element selectors:
```

Comment:
I'm not sure we need this list. It's one more place to update a list, something we're trying to move away from with the list initiative. Also, it's not like we can have the last list for known properties :)